app/default_overrides.go

```diff
@@ -257,6 +257,7 @@ func DefaultConsensusConfig() *tmcfg.Config {
 	// Set broadcast timeout to be 50 seconds in order to avoid timeouts for long block times
 	cfg.RPC.TimeoutBroadcastTxCommit = 50 * time.Second
 	cfg.RPC.MaxBodyBytes = int64(8388608) // 8 MiB
+	cfg.RPC.GRPCListenAddress = "tcp://0.0.0.0:9098"
```

Comment:
as a heads up, this will only take affect when you do `celestia-appd init` (creating a new config file with this), right?

If you have an existing state directory e.g. `~/.celestia-appd/` and your `config.toml` defines this as an empty string, I think this will get overridden by `interceptConfigs()` -> https://github.com/celestiaorg/cosmos-sdk/blob/release/v0.50.x-celestia/server/util.go#L274-L279